bedrock/externalpages/templates/externalpages/pocket/pocket-hits-signup.html

```diff
@@ -0,0 +1,61 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% block page_title %}Hits Newsletter Signup{% endblock %}
```

Comment:
**oops, i made a mistake:** Just realized that the original site has `Pocket: Pocket Hits..` as the title.. Maybe we should put the word `Pocket` back here? 😅